### DISCUSSION TITLE
MAME 0.273

### DISCUSSION THREAD

**Discussion by cuavas** [mamedev/discussions#172] [Announcements]
# [MAME 0.273](https://www.mamedev.org/?p=545)

It looks like MAME 0.273 has made it just in time for the end of 2024!  First of all, if you want to build MAME with Microsoft’s IDE, you’ll need to switch to Visual Studio 2022 and ensure you have an up-to-date version of the Visual C++ compiler.  (There’s lots more useful information about building MAME from source code on [the relevant documentation page](https://docs.mamedev.org/initialsetup/compilingmame.html).)

Microcontrollers for Sun Electronics Kangaroo and Data East E.D.F.: Earth Defense Force have been dumped and emulated.  This fixes several issues where the simulation was incorrect for Kangaroo, affecting gameplay and sound.  Speaking of which, there have been quite a few fixes for sound issues in arcade games this month.

Regional variants of the Apple II computer line allowed the user to switch between US English mode and local mode, affecting the display character set and the keyboard layout.  This release greatly improves support for language selection and adds support for several European Apple IIe and Apple IIc variants.  In other Apple emulation news, the ’030-based PowerBook series is coming to life: you might want to try out the PowerBook 140, 160, 170 and 180 or variants thereof.

There’s lots more in this release, including more Amiga sound and video cards, CPU emulation fixes, and better DMA behaviour for emulated Sound Blaster cards.  You can read about all the exciting developments in the [whatsnew.txt file](https://www.mamedev.org/releases/whatsnew_0273.txt), or get the source code and 64-bit Windows binary packages from [the download page](https://www.mamedev.org/release.html).

[Read the rest of this entry »](https://www.mamedev.org/?p=545#readmore)

**Comment by Gently28** [comment]
非常感谢 cuavas，您对DIP-SWITCHES的修正以及添加的图片非常棒，让游戏体验极佳！！！
<img width="1280" alt="1" src="https://github.com/user-attachments/assets/33fcce49-e747-4548-882a-f0b36e2a141d" />
<img width="1280" alt="2" src="https://github.com/user-attachments/assets/c03ab81d-4d3a-4d23-8240-c86ab643ce46" />

**Reply by cuavas** [reply]
Can you help me read this text from a poorly scanned Hanafuda Hana Ginga manual?
![image](https://github.com/user-attachments/assets/6912dbc1-08d6-4f44-8ec6-21754b823bae)
I think the setting name starts with “五光、四光発生” but what are the other two characters?

(The setting values are “せまい”, “ややせまい”, “ややひろい”, “ひろい” – that’s easy enough to read.)

**Reply by cuavas** [reply]
Never mind, I realised it’s “五光、四光発生間隙”.

**Reply by Gently28** [reply]
五光、四光発生間隔

**Comment by cuavas** [comment]
Bob Zed’s video highlighting some of the work in this release:
[![MAME 273 - What's new](https://img.youtube.com/vi/3IIZP90KzGg/0.jpg)](https://www.youtube.com/watch?v=3IIZP90KzGg)
